test/sequential/sequencerStaking.rules.test.ts

```diff
@@ -307,7 +308,7 @@ describe("sequencerStaking", () => {
     let cancel = await Sudo.asSudoFinalized(
       Sudo.sudoAs(
         sequencer,
-        await Rolldown.cancelRequestFromL1(chain, reqIdCanceled),
+        await Rolldown.cancelRequestFromL1(chain, disputeEndBlockNumber1),
```

Comment:
I think that the requestID represent both, requestID and when the dispute will end, TBH, I think both are fine